src/core/symbology-ng/qgsrendererv2.h

```diff
@@ -253,8 +253,15 @@ class CORE_EXPORT QgsFeatureRendererV2
     static QgsFeatureRendererV2* loadSld( const QDomNode &node, QGis::GeometryType geomType, QString &errorMessage );
 
     //! used from subclasses to create SLD Rule elements following SLD v1.1 specs
-    virtual void toSld( QDomDocument& doc, QDomElement &element ) const
-    { element.appendChild( doc.createComment( QString( "FeatureRendererV2 %1 not implemented yet" ).arg( type() ) ) ); }
+    Q_DECL_DEPRECATED virtual void toSld( QDomDocument& doc, QDomElement &element ) const
+    {
+      toSld( doc, element, QgsStringMap() );
+    }
+    virtual void toSld( QDomDocument& doc, QDomElement &element, QgsStringMap props = QgsStringMap() ) const
```

Comment:
I was just wondering how this is handled. I would probably have introduced the new method without default value for the new parameter to avoid confusion.